lib/resources/r4/related_person.yaml

```diff
@@ -75,4 +76,313 @@ fields:
     terminology:
       - display: Relationship Level
         system: http://hl7.org/fhir/resource-types
-        info_link: https://www.hl7.org/fhir/valueset-resource-types.html
\ No newline at end of file
+        info_link: https://www.hl7.org/fhir/valueset-resource-types.html
+
+- name: resourceType
+  required: 'Yes'
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "RelatedPerson"
+    }
+  note: resourceType must be RelatedPerson.
+  url: http://hl7.org/fhir/R4/json.html#resources
+
+- name: relationship
+  required: 'Yes'
+  type: CodeableConcept
+  action: create
+  description: The nature of the relationship between a patient and the related person.
+  example: |
+    {
+      "relationship": [
+        {
+          "coding": [
+            {
+              "system": "https://fhir.cerner.com/a758f80e-aa74-4118-80aa-98cc75846c76/codeSet/351",
+              "code": "1152",
+              "display": "child"
+            }
+          ]
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li>The <code>relationship.coding</code> must contain exactly one proprietary system.</li>
+      <li>The <code>relationship.coding</code> cannot contain a standard system.</li>
+    </ul>
+
+- name: patient
+  required: 'Yes'
+  type: Reference (Patient)
+  action: create
+  description: Patient that the individual is related to for the encounter.
+  example: |
+    {
+      "patient": {
+        "reference": "Patient/1626016"
+      }
+    }
+
+- name: name
+  required: 'Yes'
+  type: List of HumanName
+  description: The name associated with the individual.
+  action: create
+  example: |
+    {
+      "name": [
+        {
+          "use": "official",
+          "family": "Doe",
+          "given": [
+            "John",
+            "Smith"
+          ],
+          "period": {
+            "start": "2010-05-17T14:54:31.000Z"
+          }
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Only one name <b>must</b> be provided. The provided name must meet the following conditions:
+        <ul>
+          <li> Must have a `use` of 'official'. </li>
+          <li> Must not have an `end` date on `period`, or must not include `period`. </li>
+          <li> Must include both `given` and `family`. </li>
+          <li> Must contain either the `given` or `family` fields. </li>
```

Comment:
Given and family can't both be required and also require at least one or the other.